.github/workflows/pages-deployment.yaml

```
@@ -0,0 +1,22 @@
+on: [push]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      deployments: write
+    name: Deploy to Cloudflare Pages
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build
+        run: make site/generate
+      - name: Publish
+        uses: cloudflare/pages-action@v1
+        with:
+          branch: main
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          projectName: awesome-tech-lead
```

```suggestion
          projectName: ${{ vars.CLOUDFLARE_PAGES_PROJECT_NAME }}
```

Veja https://github.com/tech-leads-club/awesome-tech-lead/settings/variables/actions